src/panels/config/person/dialog-person-detail.ts

```diff
@@ -131,6 +120,37 @@ class DialogPersonDetail extends LitElement {
     `;
   }
 
+  private _getEntitiesPicker() {
+    if (this._deviceTrackers.length === 0) {
```

Comment:
`this. _deviceTrackers` are the device trackers that are selected for this person, not the device_trackers that are available in Home Assistant.

To get the device trackers available in Home Assistant you would do:
`Object.keys(this.hass.states).some((entityId) => entityId.substr(0, entityId.indexOf(".")) === "device_tracker");`

And as that is pretty expensive to do on every render, we should wrap that in a `memoizeOne` function, so that it will only recalculate if something the function needs changes.